app.js

```diff
@@ -39,10 +60,100 @@ export default class App {
         app.use(cookieParser);
         app.use(queryParser);
 
-        app.use('/api/products', jwt.authCustom, productRouter);
-        app.use('/api/users', jwt.authCustom, userRouter);
+        //app.use('/api/products', jwt.authCustom, productRouter);
+        //app.use('/api/users', jwt.authCustom, userRouter);
         app.use('/auth', authRouter);
 
+        app.use('/api/products', productMongoRouter);
+        app.use('/api/users', userMongoRouter);
+        app.use('/api/cities', cityMongoRouter);
+
         app.listen(port, cb);
     }
+
+    importProducts() {
+        readFile('./data/products.json')
+            .then(value => JSON.parse(value.toString()))
+            .then(values => values.map(value => {
+                Product.count({id: value.id}).then(count => {
+                    if (count === 0) {
+                        Product.create(value).then(product => {
+                            if (value.reviews) {
+                                value.reviews.map(review => {
+                                    Review.count({id: review.id}).then(count => {
+                                        if (count === 0) {
+                                            product.createReview({
+                                                id: review.id,
+                                                description: review.description
+                                            });
+                                        } else {
+                                            console.log("Record with id: %s already exists", review.id);
+                                        }
+                                    });
+                                });
+                            }
+                            return product;
+                        }).then(product => console.log("Product record created. Id: %s", product.id));
+                    } else {
+                        console.log("Record with id: %s already exists", value.id);
+                    }
+                });
+            }))
+            .catch(console.error);
+    }
+
+    importUsers() {
+        readFile('./data/users.json')
+            .then(value => JSON.parse(value.toString()))
+            .then(values => values.map(value => {
+                User.count({id: value.id}).then(count => {
+                    if (count === 0) {
+                        User.create(value)
+                            .then(user => console.log("User record created. Id: %s", user.id));
+                    } else {
+                        console.log("Record with id: %s already exists", value.id);
+                    }
+                });
+            }))
+            .catch(console.error);
+    }
+
+    importCitiesNative() {
+        let database;
+        MongoClient.connect(config.mongo.url)
+            .then(db => {
+                database = db;
```

Comment:
if there is an exception in `.connect()` database variable will always be `undefined`.
```javascript
.finally(() => {
    if (database) {
          database.close()
    }
})
```